app/src/main/java/ingsw/group1/findmyphone/activity/ActivityConstants.java

```diff
@@ -0,0 +1,10 @@
+package ingsw.group1.findmyphone.activity;
+
+/**
+ * Contains the constants that need to be shared between activities of the application,
+ * all of them are defined package-private
+ */
+public final class ActivityConstants {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'ActivityConstants' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929934)